### ISSUE TITLE
Document how to delete all tickets in Open Match

### ISSUE THREAD

**Issue by sawagh** [googleforgames/open-match#1136]
We have seen a common pattern (typically when a Frontend crashes) where there is a desire to cleanup Open Match state. While Open Match does not provide a 'delete all' option, It does have recommendations based on some APIs (QueryTicketIDs) on how to delete all Tickets. 

It will be useful to have the Open Match documentation state how this can be done with existing APIs to simplify it for the users. 

See: https://github.com/googleforgames/open-match/issues/764

**Comment by Laremere** [comment]
Hey, assigned and pending tickets are the exception to this.

Assigned will take up memory, though we want them to expire soon anyways.  So maybe that's fine.

However pending tickets will return to the pool which isn't good.

**Comment by sawagh** [comment]
Agreed. I think my example for the pattern was too specific (frontend crash). 

Through this issue however, I simply intended to provide documentation for - **How do I delete all Tickets from Open Match?** since that question has been asked a few times now. 

The frontend behaviors cleanup and handling assigned / pending seems more for a separate ecosystem focused documentation.

**Comment by yfei1** [comment]
> Agreed. I think my example for the pattern was too specific (frontend crash).
> 
> Through this issue however, I simply intended to provide documentation for - **How do I delete all Tickets from Open Match?** since that question has been asked a few times now.
> 
> The frontend behaviors cleanup and handling assigned / pending seems more for a separate ecosystem focused documentation.

I think this along with the backfill patterns should live in the FAQ section of our website, which doesn't exist at this point yet. Does Github support pinning an issue on the top of the issue page? If so, we could open a long-live issue to gather and track what are the common questions and triage them respectively instead of bringing them up in each release.

**Comment by sawagh** [comment]
This will be a documentation of using the API surface introduced in:
https://github.com/googleforgames/open-match/issues/766
